sdk/resourcemanager/Azure.ResourceManager/src/autorest.md

```diff
@@ -269,6 +269,14 @@ directive:
   - rename-model:
       from: ProviderRegistrationRequest
       to: ProviderRegistrationOptions
+  - from: resources.json
+    where: $.definitions.Alias
+    transform:
+      $["x-ms-client-name"] = "ResourceTypeAlias";
+  - from: policyDefinitions.json
+    where: $.definitions.ParameterDefinitionsValue
+    transform:
+      $["x-ms-client-name"] = "ArmPolicyParameter";
```

Comment:
Do we have similar recommendation for the things in policy assignment?
Like this: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/resources/resource-manager/Microsoft.Authorization/stable/2021-06-01/policyAssignments.json#L800
We might want it to be `ArmPolicyParameterValue`